.devcontainer/devcontainer.json

```diff
@@ -22,6 +22,7 @@
                 "vstirbu.vscode-mermaid-preview",
                 "SimonSiefke.svg-preview",
                 "eamodio.gitlens",
+                "hediet.vscode-drawio",
```

Comment:
Is this used yet?
Let's not add it until it is used.  Adding a dependency makes it harder to maintain.